src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -954,7 +914,59 @@ namespace Uno.UI {
 				src = src.parentElement;
 			}
 
-			return `${evt.pointerId};${evt.clientX};${evt.clientY};${(evt.ctrlKey ? "1" : "0")};${(evt.shiftKey ? "1" : "0")};${evt.buttons};${evt.button};${evt.pointerType};${srcHandle};${evt.timeStamp};${evt.pressure}`;
+			let pointerId: number, pointerType: string, pressure: number;
+			let wheelDeltaX: number, wheelDeltaY: number;
+			if (evt instanceof WheelEvent) {
+				pointerId = (evt as any).mozInputSource ? 0 : 1; // Try to match the mouse pointer ID 0 for FF, 1 for others
+				pointerType = "mouse";
+				pressure = 0.5; // like WinUI
+				wheelDeltaX = evt.deltaX;
+				wheelDeltaY = evt.deltaY;
+
+				switch (evt.deltaMode) {
+					case WheelEvent.DOM_DELTA_LINE: // Actually this is supported only by FF
+						wheelDeltaX *= this.WheelLineSize;
+						wheelDeltaY *= this.WheelLineSize;
+						break;
+					case WheelEvent.DOM_DELTA_PAGE:
+						wheelDeltaX *= document.documentElement.clientWidth;
+						wheelDeltaY *= document.documentElement.clientHeight;
+						break;
+				}
+			} else {
+				pointerId = evt.pointerId;
+				pointerType = evt.pointerType;
+				pressure = evt.pressure;
+				wheelDeltaX = 0;
+				wheelDeltaY = 0;
+			}
+
+			return `${pointerId};${evt.clientX};${evt.clientY};${(evt.ctrlKey ? "1" : "0")};${(evt.shiftKey ? "1" : "0")};${evt.buttons};${evt.button};${pointerType};${srcHandle};${evt.timeStamp};${pressure};${wheelDeltaX};${wheelDeltaY}`;
```

Comment:
We should find a way to use data structures instead of a parsed string here.

Reply:
https://github.com/unoplatform/uno/pull/3003/files/9c4cff69ff2acf77fa1233c3a0cae020f4344bb2#diff-a1ec1a13fbf77968165020e117b951faR367 already documented and bug already open :)